### DISCUSSION TITLE
Moving files from MergerFS to NAS

### DISCUSSION THREAD

**Discussion by PooreOne** [trapexit/mergerfs#1335] [General]
I currently have a Plex server with MergerFS running fine on Ubuntu 22.04 server. I want to move everything from the MergerFS pool to the NAS. What's the best way to move the files over? I would prefer not to do it all at once but slowly.

**Comment by trapexit** [comment]
Sorry but not sure what you're asking or how mergerfs is involved. Can you not just copy the files? rsync or something?
